addons/opacity-slider/addon.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Opacity slider",
+  "description": "Adds opacity slider to the costume editor color picker.",
+  "tags": ["editor", "costumeEditor", "recommended"],
```

Comment:
Remove `recommended` tag
Add `beta` tag
(?)
I have my doubts we found all bugs

Reply:
Of course the "recommended" tag should be added on v1.29 release

Reply:
If `recommended` tag is removed, `featured` tag should be added

Reply:
Do `featured` and `beta` work simultaneously?